packages/ui-toolkit/src/components/atoms/Toaster/state.ts

```diff
@@ -0,0 +1,153 @@
+import type {
```

Comment:
Can create a `constants` folder and move `state.ts` and `type.ts` files within it.

Reply:
state.ts file contains logical part which included how different kind of toast added to dom and exported main toast method which will be used to trigger toast, so including this in constants folder will mislead to developers. 
while types files simply contains type definations and self explained by file name and no need to more abstractions.

Reply:
ack

Reply:
Can we update the file name to `state.ts` I feel the name is slighlty misleading